app/controllers/feeds_controller.rb

```diff
@@ -58,6 +60,7 @@ def update
   def destroy
     respond_to do |format|
       if @feed.destroy
+        @feed.podcast&.publish!
```

Comment:
This leaves the RSS file in S3, right?  Does DTR handle that by looking at the Feeder API and returning 404s?

Reply:
Yeah, currently this doesn't do anything.

But I was thinking in the future, we may decide to "do things".  Delete the RSS from S3, or redirect, or something?  This at least sets up the hook, and brings these to parity with the API controllers (which have after-destroy hooks).